### DISCUSSION TITLE
Wallpaper (I use Rain v2) is stretched and low-res upon startup

### DISCUSSION THREAD

**Discussion by sheenygami** [rocksdanister/lively#1463] [General]
Hi! This is my first posting here. I'm just having a little issue so I had to.

The wallpaper I use is the Rain v2 which fills my dual monitor setup. The app and my 2 monitors are running on the GPU. I have also set the app to start with windows.

The issue is:
- when the app runs, the wallpaper looked stretched across the monitors (not filled but stretched), and the reso kind of drops.
- the wallpaper doesn't freeze, it just looks like a stretched low-quality wallpaper. I waited for about a minute to see if it adjusts but it didn't.

What I did is:
- exit the app
- when the app closes, there's a solid magenta "wallpaper" that displays (which is not my default wallpaper -- sometimes it stays, sometimes it doesn't)
- then run the app again
- when it runs, the Rain v2 then returns to its hi-res, non-stretched state

**Comment by rocksdanister** [comment]
The included backgrounds for Rain are just standard 1080p images and its not meant for dual monitor or non 16:9 aspect ratios.

Add your own correct size image from the customise menu.

**Reply by sheenygami** [reply]
Thank you for the response!

I'll keep observing and if keeps doing it, I'll change the background.
